docs/UserGuide.md

```diff
@@ -212,6 +205,12 @@ Parameter      | Description                   | Example
 
 </div>
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** The dashboard’s Schedule will be updated accordingly.
```

Comment:
I think this should belong in the existing notebox above, not in a separate one.